### PR TITLE
Added methods :meth:`~.get_lines` and :meth:`~.get_value` to :class:`~.Angle`

### PR DESCRIPTION
**Pull request by Animfysyk**
## Overview: What does this pull request change?
It adds 2 methods for Angle : one to access lines that form an angle (it's not very usefull, just to make it clean), and another that returns the value of the angle in radians or degrees (boolean parameter), respecting the way the angle is plotted, including the 'other_angle' parameter when creating an angle.

## Motivation and Explanation: Why and how do your changes improve the library?

I don't really know if there is already a function that does it ... In fact, someone from the discord server told me to code it and make a PR when I asked a question about angles. That's why I did it, it's my first time trying to contribute.

## Links to added or changed documentation pages
 
I didn't make any documentation for this, since the name of the method is quite explicit. Tell me if there is something else I can do to improve this PR.

## Further Information and Comments

I tested it quite a lot, it worked as I wanted. However, if you detect any issue, feel free to tell me what's wrong. There are some cases where this method fails, when the angle is 0 or pi, and it's an issue on Angle class, manim says "Lines do not intersect". This could be the subject of a future PR.

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [x] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by PhotonSpheres** [comment]
Thank you for your effort, those are some great ideas to implement!

`get_lines`:
I feel like it would be more in the spirit of the rest of the library if the two lines would be returned in a `VGroup` instead as two single mobjects. What do you think?

`get_value`:
- I like the idea of the `degrees` parameter!
- Wouldn't it be easier to just safe the `angle_fin` value to the `Angle` mobject and return this number when `get_value` gets called instead of calculating the angle again? 

After this is sorted out we might wanna look into type hints and doc strings.

Again, great effort for your first PR.

**Comment by PhotonSpheres** [comment]
Thanks for the improvements! I think it might be a bit cleaner to only safe the final angle once to `self`, maybe like this:

```py
             if not other_angle:
                start_angle = angle_1
                if angle_2 > angle_1:
                    angle_fin = angle_2 - angle_1
                else:
                    angle_fin = 2 * np.pi - (angle_1 - angle_2)
            else:
                start_angle = angle_1
                if angle_2 < angle_1:
                    angle_fin = -angle_1 + angle_2
                else:
                    angle_fin = -2 * np.pi + (angle_2 - angle_1)

            self.angle_fin = angle_fin
```

This should do the same with less code.

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -3015,6 +3019,55 @@ def __init__(
 
         self.set_points(angle_mobject.get_points())
 
+    def get_lines(self) -> VGroup:
+        """Get the lines forming an angle of the :class:`Angle` class.
+
+        Returns
+        -------
+        :class:`VGroup`
```
Comment (PhotonSpheres) [inline manim/mobject/geometry.py:3027]:
```suggestion
        :class:`~.VGroup`
```

File: manim/mobject/geometry.py [hunk 2]
```diff
@@ -3015,6 +3019,55 @@ def __init__(
...
+            The VGroup of the lines that form an angle of the :class:`Angle` class.
```
Comment (PhotonSpheres) [inline manim/mobject/geometry.py:3028]:
```suggestion
            A :class:`~.VGroup` containing the lines that form the angle of the :class:`Angle` class.
```

File: manim/mobject/geometry.py [hunk 3]
```diff
@@ -3015,6 +3019,55 @@ def __init__(
...
+
+        return VGroup(self.line1, self.line2)
+
+    def get_value(self, degrees: bool = False) -> float:
+        """Get the value of an angle of the :class:`Angle` class.
+
+        Parameters
+        ----------
+        degrees
+            A boolean to decide the unit (deg/rad) in which the value of the angle is given.
```
Comment (PhotonSpheres) [inline manim/mobject/geometry.py:3048]:
```suggestion
            A boolean to decide the unit (deg/rad) in which the value of the angle is returned.
```

**Review by PhotonSpheres — CHANGES_REQUESTED** [review]
Only some small suggestions.

File: manim/mobject/geometry.py [hunk 4]
```diff
@@ -3015,6 +3019,62 @@ def __init__(
...
+
+            class GetValueExample(Scene):
+                def construct(self):
+                    line1 = Line(LEFT+(1/3)*UP, RIGHT+(1/3)*DOWN)
+                    line2 = Line(DOWN+(1/3)*RIGHT, UP+(1/3)*LEFT)
+
+                    angle = Angle(line1, line2, radius=0.4)
+
+                    value = DecimalNumber(angle.get_value(degrees=True), unit="^{\\circ}")
+                    value.move_to(Angle(line1, line2, radius = 0.4 + 3*SMALL_BUFF).point_from_proportion(0.5))
```
Comment (PhotonSpheres) [inline manim/mobject/geometry.py:3069]:
```suggestion
                    value.next_to(angle, UR, buff=0)
```
As seen here: https://manimce--1881.org.readthedocs.build/en/1881/reference/manim.mobject.geometry.Angle.html#manim.mobject.geometry.Angle , the number crosses with the angle which makes for an anesthetic look. I still feel the much easier version above works and looks better.

**Review by PhotonSpheres — CHANGES_REQUESTED** [review]
If this last issue is resolved this PR is good to go for me!

**Review by PhotonSpheres — APPROVED** [review]
LGTM! Thanks again for your contribution!

**Review by behackl — APPROVED** [review]
This looks good to me. I left some comments about the currently designated attribute names and how I think they could be improved.

Apart from a brief discussion about the attribute names, this can be merged.

**Pull request merged**